src/sql/add_data.sql

```diff
@@ -0,0 +1,17 @@
+insert into products (title, description, image, price) values 
+('Plush bunting', 'Baby Boys and Girls Plush Bunting', 
+'https://slimages.macysassets.com/is/image/MCY/products/3/optimized/13987513_fpx.tif', 235),
+('Carters PJ', 'Baby Boy or Girl Christmas Snap-Up Thermal Footless PJs', 
+'https://slimages.macysassets.com/is/image/MCY/products/3/optimized/17716933_fpx.tif', 616),
+('Hudson Baby', 'Cotton Union Suit, 2-Pack', 
+'https://slimages.macysassets.com/is/image/MCY/products/1/optimized/11879221_fpx.tif', 708),
+('Jordan Bodysuit', 'Baby Boys Jumpman Cotton Bodysuit', 
+'https://slimages.macysassets.com/is/image/MCY/products/8/optimized/17268678_fpx.tif', 369);
+
+
+insert into stocks (product_id, "count") values
+('f00ca216-43f2-4988-ac2e-d39902e56390', 4),
```

Comment:
Could you please elaborate? As far as I understand, I have to provide specific data records to populate the db, correct?
Or do you mean finding this id dynamically via select as a subquery? and then doing the insert?

Reply:
You inserted products on the previous request, so you already have ID's of these products. Instead of manually put IDs here, you can select them from the products table and put here

Reply:
Got it, thanks! Agree that it's a better option (probably I could search by title and get id for insert).